### PR TITLE
Update client library to target cma < 1.3.0, update pypi targets to py3

### PR DESCRIPTION
**Pull request by Jkovarik**
Minor PR to update pypi targets to python 3, update circleci configuration to match.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 __version__ = imp.load_source('version', 'version.py').__version__
 
 # Arguments marked as "Required" below must be included for upload to PyPI.
-# Fields marked as "Optional" may be commented out.
+# Fields marked asF "Optional" may be commented out.
```
Comment (Jkovarik) [inline setup.py]:
```suggestion
# Fields marked as "Optional" may be commented out.
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -4,9 +4,9 @@
 #
 version: 2
 references:
-  container_python27: &container_python27
+  container_python27: &container_python38
```
Comment (nemreid) [inline .circleci/config.yml]:
```suggestion
  container_python38: &container_python38
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -32,11 +32,11 @@ references:
         - v1-dependencies-{{ checksum "requirements.txt" }}-{{ checksum "requirements-dev.txt" }}
         # fallback to using the latest cache if no exact match is found
         - v1-dependencies-
-  
-  save_cache_python27: &save_cache_python27
+
+  save_cache_python27: &save_cache_python38
```
Comment (nemreid) [inline .circleci/config.yml]:
```suggestion
  save_cache_python38: &save_cache_python38
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
+## Unreleased
+
+- **CUMULUS-1486**
+  - Updated target python versions to more accurately reflect python 3 compatibility/cumulus-message adapter versions
+  - Updated requirements to require cumulus-message-adapter 1.3.x
```
Comment (nemreid) [inline CHANGELOG.md]: .. But it requires `<1.3.x`, not `<=1.3.x`, right?

Comment (Jkovarik) [reply CHANGELOG.md]:
```suggestion
  - Updated requirements to require cumulus-message-adapter >= 1.1.3, < 1.3.x
```

**Pull request merged**